client/webserver/site/src/js/app.js

```diff
@@ -385,9 +386,15 @@ export default class Application {
    * --------   -----------
    * notes      Notifications of all levels.
    * book       Order book feed.
+   * ws.........Websocket connection status changes.
    */
   log (loggerID, ...msg) {
-    if (this.loggers[loggerID]) console.log(`[${loggerID}]:`, ...msg)
+    const stamp = new Date()
+    const h = stamp.getHours().toString().padStart(2, '0')
+    const m = stamp.getMinutes().toString().padStart(2, '0')
+    const s = stamp.getSeconds().toString().padStart(2, '0')
+    const ms = stamp.getMilliseconds().toString().padStart(3, '0')
+    if (this.loggers[loggerID]) console.log(`${`${h}:${m}:${s}:${ms}`}[${loggerID}]:`, ...msg)
```

Comment:
Maybe move this check up before declaring variables?
`if (!this.loggers[loggerID])) return;`